create/faq.md

```diff
@@ -189,4 +176,50 @@ See our [contact page](/learn/what_is_meilisearch/contact.md).
 
 ## I have just updated MeiliSearch, and I am getting an error: "Cannot open database, expected MeiliSearch engine version..."
 
-MeiliSearch minor versions are not compatible with each other because the way we represent data internally changes with the addition of new breaking features. To fix this, it suffices to delete your database folder (`data.ms` by default) and re-index your documents with the new version.
+Until our first stable release (v1.0), MeiliSearch minor versions are not compatible with each other, i.e. **every new version is considered breaking** with the small exception of bug-fixing patches. To fix this error, simply delete your database folder (`data.ms` by default) and re-index your documents with the current-version engine. See [updating MeiliSearch](/reference/features/installation.md#updating-meilisearch) for more information.
+
+## What are the recommended requirements for hosting a MeiliSearch instance?
+
+**Short answer**: It's impossible to say. We recommend [experimenting with RAM availability and database size](#how-does-hardware-affect-search-speed) to find the hardware + relevancy setup that works best for your dataset.
+
+**Long answer**: The disk size and RAM usage of your MeiliSearch database can vary widely. The following factors have a great effect on space and memory use:
+
+- The number of documents
+- The size of documents
+- The number of indexed fields
+- The number of faceted fields
+- The size of each update
+- The number of different words present in documents (beware heavily multi-lingual datasets and fields with unique words, such as IDs)
```

Comment:
```suggestion
- The number of different words present in documents (beware heavily multi-lingual datasets and indexed fields with unique words, such as IDs or URLs)
```

Reply:
It's not a necessary change until you already talk about that on the last line of the page.